src/Illuminate/Foundation/helpers.php

```diff
@@ -470,6 +470,27 @@ function factory()
     }
 }
 
+if (! function_exists('has_errors')) {
+    /**
+     * Returns true if session has errors, and optionally key-specific errors.
+     *
+     * @param  string|null  $key
+     * @param  string|null  $bag
+     * @return bool
+     */
+    function has_errors($key = null, $bag = null)
```

Comment:
`Null` is inconsistent with other parts of the framework. I think it should stay as `default` https://github.com/laravel/framework/blob/5.4/src/Illuminate/Support/ViewErrorBag.php#L23